test/poll.go

```diff
@@ -0,0 +1,26 @@
+package test
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+// Poll repeatedly evaluates condition until we either timeout, or it succeeds.
```

Comment:
This comment is not correct. `Poll` doesn't "evaluate condition", but calls a function and waits until it returns expected value, or times out.